docs/reference-implementation/Troubleshooting.md

```diff
@@ -74,6 +74,26 @@ Location: SwedenCentral, Current Limit: 100, Current Usage: 96, Additional Requi
 
 **Error:**
 
+```console
+│ Agent Pool Name: "workloadpool"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with response: {
+│   "code": "SubscriptionNotEnabledEncryptionAtHost",
+│   "details": null,
+│   "message": "Subscription does not enable EncryptionAtHost.",
+│   "subcode": ""
+│  }
+```
+
+**Description:** Host encryption needs to be enabled at subscription-level when `aks_enable_host_encryption` is set to `true` in `variables-<env>.tf`.
```

Comment:
If we really need to make host encryption a requirement, we'd need to put it into the getting started guide. Otherwise every single customer will likely fail over this. 
Isn't there some way we can enable this for us without enabling it for everybody who would clone the repo?

Reply:
@sebader I completely forgot this PR. I don't see a way to only enable it for us. The auto-registration via TF is also not an option as it would require us to define all resource provider registrations. I'm ok to add it to the getting started guide though. If that's an acceptable solution.